script/install.sh

```diff
@@ -1,30 +1,34 @@
-#!/usr/bin/env bash
-set -euo pipefail
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```

so the /bin/sh path is not hardcoded

Reply:
Thanks! From some reading it seems like both are "about as portable" as each other. Have you had experiences where they're not?